pandas/core/generic.py

```diff
@@ -9660,7 +9660,8 @@ def last(self, offset) -> Self:
         Notes
         -----
         .. deprecated:: 2.1.0
-            Please create a mask and filter using `.loc` instead
+            Please create a mask and filter using `.loc` instead.
+            See examples for both cases.
```

Comment:
Can you just write `See examples below.` instead? In the same line probably better.